test/e2e/walkthrough.go

```diff
@@ -192,6 +193,40 @@ var _ = framework.ServiceCatalogDescribe("walkthrough", func() {
 		err = util.WaitForInstanceToNotExist(f.ServiceCatalogClientSet.ServicecatalogV1alpha1(), testnamespace.Name, instanceName)
 		Expect(err).NotTo(HaveOccurred())
 
+		By("Creating a ServiceInstance using a default plan")
```

Comment:
This specific test is meant for the ups-broker walkthrough, so I think we should probably move this piece to a different location. But it's not hurting anything and is a good test, and I know the rebase pains for this PR are immense, so I think that can be addressed in a followup!